wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/jpa/JpaWorkspaceDao.java

```diff
@@ -159,6 +161,26 @@ public WorkspaceImpl get(String name, String namespace) throws NotFoundException
         }
     }
 
+    @Override
+    @Transactional
+    public List<WorkspaceImpl> getWorkspaces(boolean isTemporary, int skipCount, int maxItems) throws ServerException {
```

Comment:
IMO it;s a huge overhead for a method which results doesn't supposed to return via rest.